appmanager.ipynb

```diff
@@ -76,12 +76,17 @@
     "        if app_select.value:\n",
     "            display_app_box(app_select.value)\n",
     "\n",
-    "app_select = ipw.Select()\n",
-    "update_app_list()\n",
+    "app_select = ipw.Select(layout={'height': '{}px'.format(len(registry)*20)})\n",
     "app_select.observe(on_app_change, names='value')\n",
     "\n",
     "out_app = ipw.Output()\n",
-    "display(app_select, out_app)"
+    "out_update = ipw.Output()\n",
```

Comment:
Can you reuse the other output?
Currently, your output stays, even if one selects another app afterwards.

Reply:
✓